code/modules/mob/living/simple_animal/hostile/giant_spider/base_spider.dm

```diff
@@ -166,9 +166,9 @@
 			client.darkness_planemaster.alpha = 100
 		client.color = list(
 					1,0,0,0,
-					0,1,0,0,
-	 				0,0,1,0,
-		 			0,0,-0.1,1,
+					0,1.3,0,0,
+	 				0,0,1.3,0,
+		 			0,-0.3,-0.3,1,
```

Comment:
why are these changes going on in this PR?

Reply:
I don't know, I don't remember changing these.